### PR TITLE
O3 1314 - Update the configurations to local instance test backend for 3.X E2E test

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose
The pull request is to update configurations and readme guidance for the local instance test backend and this pull request is related to the issue ticket number [O3-1314](https://issues.openmrs.org/browse/O3-1314).

## Goals
- Change the test backend to a local instance for 3.x E2E testing

## Approach
1. Update configurations for local instance test backend for 3.X E2E test
2. Update the readme guidance to set up the local environment.

#### Passed test (Login)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,9 +130,39 @@ MySQL password should be the same for initialSetupTests as openmrs password
...
+   
+2. Package the distribution and prepare the run
+    ```
+    mvn clean package
+    ```
+   
+### If the build fails at this point should be destroyed the associated volumes
+```
+cd run/docker
+docker compuse down -v
```
[the remark this answers is not in the document]
Comment (pasindur99) [reply README.md:152]: Thank you @dkayiwa I will update it to **compose**.

File: README.md [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
-![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/TRUNK-MASTER) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/RESTWS-RESTWS) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/FHIR-FM2) | ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/SON-OPENMRSCOREMASTER)
+| [TRUNK](https://ci.openmrs.org/browse/TRUNK-MASTER)                                         | [REST API](https://ci.openmrs.org/browse/RESTWS-RESTWS)                                      | [FHIR API](https://ci.openmrs.org/browse/FHIR-FM2)                                      | [SONAQUBE DASHBOARD](https://sonar.openmrs.org/dashboard?id=org.openmrs%3Aopenmrs)                   |
+|---------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
```
Comment (kdaud) [inline README.md:6]: @pasindur99 is this change part of the ticket requirement?

Comment (pasindur99) [reply README.md:6]: @kdaud The changes are auto-generated through my IDE. I will replace it and update the pr.

Comment (pasindur99) [reply README.md]: @kdaud Fixed this with [3b7d891](https://github.com/openmrs/openmrs-contrib-qaframework/pull/429/commits/3b7d8910bd19b5ab1f56c66467d8ce965cdae305)

**Review by jayasanka-sack — APPROVED** [review]
Thanks, @pasindur99 !

**Comment by jayasanka-sack** [comment]
Merging since failing tests are independent of this PR.

**Pull request merged**